include/graphblas/utils/iscomplex.hpp

```diff
@@ -53,6 +58,22 @@ namespace grb {
 				/** Whether the type \a C is std::complex */
 				static constexpr const bool value = false;
 
+				/**
+				 * @returns The conjugate of a given value if \a C is a complex type, or
+				 *          the given value if \a C is not complex.
+				 */
+				static C conjugate( const C &x ) {
+					return x;
+				}
+
+				/**
+				 * @returns The absolute value of a given value if \a C is a complex type,
+				 *          or the given value if \a C is not complex.
+				 */
+				static C modulus( const C &x ) {
+					return x;
```

Comment:
Shouldn't modulus return absolute value even for double?

Reply:
The spec of is_complex says that functions like modulus and conjugate become no-ops the moment C is not std::complex

(For a free function utils::modulus I would agree)